README.md

```diff
@@ -173,4 +173,5 @@ This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

```

Comment:
Well done on using the professional `README.md` file but in order to make it even better 🔝 you should remove this `NOTE` as it was meant to explain to you how to configure a section of the README and not for the visitors of your repo to see. 📎

Reply:
Kindly, take this requirement as optional.

In order to have a professional README, it is recommended to remove this commented text, as mentioned be the reviewer.